dice-game-ui-angular/src/app/statistics/statistics.component.ts

```diff
@@ -1,27 +1,50 @@
 import { Component, OnInit } from '@angular/core';
 
-import { StatisticsData, StatisticsService } from '../services/statistics-service';
+import { GameStatistics, StatisticsService } from '../services/statistics-service';
+
+export class StatisticsDTO {
```

Comment:
DTOs are not needed to be classes, interface is enough to just store some data.

Reply:
Changed